src/community/geofence-server/src/test/java/org/geoserver/geofence/rest/MultiPolygonAdapterTest.java

```diff
@@ -0,0 +1,48 @@
+package org.geoserver.geofence.rest;
```

Comment:
@NielsCharlier this file should also have a copyright header (even though it is a test).